packages/evolution-backend/src/config/projectConfig.ts

```diff
@@ -52,6 +52,7 @@ export const defaultConfig: ProjectServerConfig<unknown, unknown, unknown, unkno
             google,
             responses: {
                 _isCompleted: responses?._isCompleted,
+                acceptToBeContactedByEmailForHelp: responses?.acceptToBeContactedByEmailForHelp,
```

Comment:
@tahini This is interview-specific, but I don't know how else to proceed since the types are in evolution-backend. Do you have a suggestion?

Reply:
Indeed, this is survey specific. We need to implement a way to extend the validation list. For the server side, we already have it (see serverConfiguration.ts in od_mtl_2023), but the list itself cannot add new columns. We should have a way to specify filter columns as a prop in the  InterviewListComponent, and the admin client can specify those additional columsn, with filters, etc.